src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/DescendantToken/context/defaultContext.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/Test.java b/Test.java
+index ace1b4b..e116e08 100644
+--- a/Test.java
++++ b/Test.java
+@@ -8,6 +8,7 @@ public class Test {
+             default:
+                 i--;
+                 i++;
++                System.out.println();
+                 break;
+         }
+     }
+@@ -21,7 +22,6 @@ public class Test {
+                 i++;
+                 break;
+ 
+-            default:
```

Comment:
duplicate "default:" is compiler error. We should not work on non-compiled sources.
Please udpate test to search of amount of fields in class, it will be much compact.

Reply:
done